systems/controllers.py

```diff
@@ -91,11 +93,15 @@ class ClimateController:
     """Controller class for climate sensors.
 
     Attributes:
-        weathertemperature (int): Defaults to 0.
+        outsidetemperature (int): Defaults to 0.
+        insidetemperature (int): Defaults to 0.
         fanpower (int): Defaults to 0.
+        temperaturesetting (int): Defaults to 0.
     """
-    weathertemperature: int = 0
+    outsidetemperature: int = 0
```

Comment:
@DhyeyLalseta @rjb75, would it be better to camelCase these variables or is it fine all lower case?

Reply:
Oh yeah I was thinking that too, I just decided to keep it all lowercase to match the other variables

Reply:
completely up to you all, my personal opinion i like all lowercase a bit better but no strong feelings towards any

Reply:
Hmm, yeah I think camelCase would be better. It's more readable and the exact same variable names will need to be used in the UI as well
![image](https://user-images.githubusercontent.com/14501811/111569910-47273600-8769-11eb-97d1-f0a5da72611e.png)